src/field/captcha/extended_captcha.jsx

```diff
@@ -6,16 +6,23 @@ const noop = () => {};
 
 const RECAPTCHA_V2_PROVIDER = 'recaptcha_v2';
 const RECAPTCHA_ENTERPRISE_PROVIDER = 'recaptcha_enterprise';
+const HCAPTCHA_PROVIDER = 'hcaptcha';
+const FRIENDLY_CAPTCHA_PROVIDER = 'friendly_captcha';
 
-export const isRecaptcha = provider =>
-  provider === RECAPTCHA_ENTERPRISE_PROVIDER || provider === RECAPTCHA_V2_PROVIDER;
+const AUTH0_PROVIDER = 'auth0'
 
-const getRecaptchaProvider = provider => {
+export const isExtendedCaptcha = provider => provider !== AUTH0_PROVIDER;
```

Comment:
Is `auth0` the only possible provider aport from the other 4 ?

Reply:
yes!